experimenter/experimenter/experiments/tests/test_models.py

```diff
@@ -1710,34 +1710,62 @@ def test_timeline_dates_includes_correct_status_dates_and_flags(self):
             new_publish_status=NimbusExperiment.PublishStatus.REVIEW,
             changed_on=datetime.datetime(2023, 4, 1),
         )
+
+        # Calculate expected days
+        enrollment_days = (
+            experiment.computed_enrollment_days
+            if experiment._enrollment_end_date
+            else None
+        )
+        duration_days = (
+            experiment.computed_duration_days if experiment.computed_end_date else None
+        )
```

Comment:
These should be static values, otherwise we're just computing it the same way twice and comparing.

Reply:
good point